src/repository/entity/OwnerEntity.java

```diff
@@ -0,0 +1,75 @@
+package repository.entity;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class OwnerEntity extends BaseEntity {
+
+    public OwnerEntity() {
+    }
+
+    public OwnerEntity(UUID ownerId, String name, LocalDate birthDate) {
+        super(ownerId, name, birthDate);
+    }
+
+    @Override
+    public UUID getId() {
+        return super.getId();
+    }
+
+    @Override
+    public void setId(UUID id) {
+        super.setId(id);
+    }
+
+    @Override
+    public String getName() {
+        return super.getName();
+    }
+
+    @Override
+    public void setName(String name) {
+        super.setName(name);
+    }
+
+    @Override
+    public LocalDate getBirthDate() {
+        return super.getBirthDate();
+    }
+
+    @Override
+    public void setBirthDate(LocalDate birthDate) {
+        super.setBirthDate(birthDate);
+    }
+
+    @Override
+    public String toString() {
+        return "ID: "
+                + super.getId()
+                + ", Name: "
+                + super.getName()
+                + ", Birthdate: "
+                + super.getBirthDate();
+    }
+
+    @Override
+    public int hashCode() {
+        return this.getName().length()
+                + this.getBirthDate().lengthOfMonth()
+                + this.getId().hashCode();
+    }
+
+    @Override
+    public boolean equals(Object otherOwner) {
```

Comment:
It is better to name parameter 'object', because you cannot be sure which class would be passed in equals